docs/source/library-user-guide/upgrading.md

```diff
@@ -152,6 +152,47 @@ Instead of silently succeeding.
 
 The remove API no longer requires a mutable instance
 
+### FFI object conversion
```

Comment:
Nice - I also think it would be good to mention removal of the return type functions here